src/config/index.ts

```diff
@@ -146,6 +146,7 @@ export const UPDATE_STATEMENT_VALIDATION_ERRORS_PAGE = "update-statement-validat
 export const REMOVE_SOLD_ALL_LAND_FILTER_PAGE = "remove-sold-all-land-filter";
 export const REMOVE_IS_ENTITY_REGISTERED_OWNER_PAGE = "remove-is-entity-registered-owner";
 export const REMOVE_CANNOT_USE_PAGE = "remove-cannot-use";
+export const REMOVE_NEED_MAKE_CHANGES_PAGE = "remove-need-to-make-changes-to-entity";
```

Comment:
It's not a great choice of words for a constant name but totally understand that you've tried to match the defined HTML page name. How about if you drop the `_MAKE_` so we're left with just `REMOVE_NEED_CHANGES_PAGE`?

(`REMOVE_CHANGES_REQUIRED_PAGE` would be my first choice, but then we diverge from the HTML page name)